app/controllers/merchant/orders_controller.rb

```diff
@@ -2,8 +2,9 @@ class Merchant::OrdersController < ApplicationController
   before_action :require_merchant
 
   def show
-    merchant = current_user.merchant
-    @item_orders = merchant.item_orders.where("order_id = #{params[:id]}")
+    # @merchant = current_user.merchant
+    @item_orders = current_user.merchant.item_orders.where("order_id = #{params[:id]}")
```

Comment:
It might be better to move this query into an instance method in the Merchant model.

Reply:
Great idea!